olcPixelGameEngine.h

```diff
@@ -151,7 +151,7 @@
 
 	Author
 	~~~~~~
-	David Barr, aka javidx9, �OneLoneCoder 2018, 2019, 2020
+	David Barr, aka javidx9, ©OneLoneCoder 2018, 2019, 2020

```

Comment:
wut, apparently the copyright symbol has changed, i wonder why

Reply:
I believe the editor said something about UTF-8 versus some ISO encoding. I forget specifically what it said but that's probably why the line changed.